funnel/views/commentvote.py

```diff
@@ -171,6 +171,10 @@ def new_comment(self):
                 if item.get('to'):
                     # Sender is set to None to prevent revealing email.
                     send_mail(sender=None, body=email_body, **item)
+        else:
+            # FIXME: ValidationError here. But we cannot return the error as
+            # it redirects to the proposal page, where the comment form is.
+            flash(_("Something went wrong, please try again"), category='error')
         # Redirect despite this being the same page because HTTP 303 is required to not break
         # the browser Back button
         return redirect(to_redirect, code=303)
```

Comment:
This needs to be an Ajax call. There's no need to do a full page submit anymore.